src/systems/particle_emitter/ParticleEmitter.cc

```diff
@@ -40,10 +44,36 @@ using namespace systems;
 // Private data class.
 class ignition::gazebo::systems::ParticleEmitterPrivate
 {
-  /// \brief The particle emitter.
+  /// \brief Callback for receoving particle emitter commands.
+  /// \param[in] _msg Particle emitter message.
+  public: void OnCmd(const ignition::msgs::ParticleEmitter &_msg);
+
+  /// \brief The particle emitter parsed from SDF.
   public: ignition::msgs::ParticleEmitter emitter;
+
+  /// \brief The transport node.
+  public: ignition::transport::Node node;
+
+  /// \brief Model interface.
+  public: Model model{kNullEntity};
```

Comment:
It looks to me like this is only being used in the `Configure` method, so can we make this a local variable in that method instead?

Reply:
removed. 468b02a